openshift/app.bc.yaml

```diff
@@ -48,6 +48,7 @@ objects:
         dockerfile: |-
           FROM BuildConfig
           ARG APP_ROOT=/opt/app-root/src
+          ENV NO_UPDATE_NOTIFIER=true
           ENV PATH="/usr/lib/libreoffice/program:${PATH}" \
               PYTHONUNBUFFERED=1
```

Comment:
Didn't realize this already had an ENV layer. We can reduce one docker layer in the build if we combined the lines together
```
ENV NO_UPDATE_NOTIFIER=true \
    PATH="/usr/lib/libreoffice/program:${PATH}" \
    PYTHONUNBUFFERED=1
```

Reply:
OK, updated.